operations/security_and_authentication/Add_LDAP_User_Federation.md

```diff
@@ -271,13 +275,13 @@ LDAP user federation is not currently configured in Keycloak. For example, if it
       > **NOTE:** A properly configured Docker or Podman environment is required.
```

Comment:
I think we need to remove step 1 above where it mounts /mnt/pitdata since we are no longer using that below.